.github/workflows/scripts/post_before_script.sh

```diff
@@ -6,17 +6,16 @@ if [[ "$TEST" == "upgrade" ]]; then
     exit
 fi
 
-cmd_stdin_prefix bash -c "cat > /root/sign-metadata.sh" < "$GITHUB_WORKSPACE"/pulp_rpm/tests/functional/sign-metadata.sh
+cmd_prefix bash -c "mkdir /var/lib/pulp/scripts"
+cmd_prefix bash -c "chown pulp:pulp /var/lib/pulp/scripts"
```

Comment:
My experiments with the container in the cli suggest that it is not really necessary to make the directory owned by pulp.